### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by andborges**

### CODE REVIEW

**Review by Peixer — APPROVED** [review]

File: packages/extension/src/ui/action/App.vue [hunk 1]
```diff
@@ -331,8 +334,32 @@ const setNetwork = async (network: BaseNetwork) => {
     }
   }
 };
+
+const onUpdatedBalance = async () => {
+  try {
+    const thisNetworkName = currentNetwork.value.name;
```
Comment (Peixer) [inline packages/extension/src/ui/action/App.vue:340]: @andborges just a small thing, rename this variable, example `currentNetworkName`

File: packages/extension/src/providers/kadena/ui/send-transaction/index.vue [hunk 2]
```diff
@@ -8,6 +8,14 @@
...
 
+      <p
+        class="send-transaction__description"
+        style="color: red"
+        :class="{ popup: isPopup }"
+      >
+        {{ errorMsg }}
+      </p>
+
       <send-address-input
```
Comment (Takadenoshi) [inline packages/extension/src/providers/kadena/ui/send-transaction/index.vue]: For the error messaging, can we use a `send-alert` component & the styling that other implementations use?

e.g. ./src/providers/bitcoin/ui/send-transaction/components/send-alert.vue

**Review by Takadenoshi — CHANGES_REQUESTED** [review]

File: packages/extension/src/providers/kadena/ui/send-transaction/index.vue (hunk 2 above)
Comment (andborges) [reply packages/extension/src/providers/kadena/ui/send-transaction/index.vue]: Hi @Takadenoshi Nice catch!

I made the changes, can you check if it's ok now? Tks

Comment (Takadenoshi) [reply packages/extension/src/providers/kadena/ui/send-transaction/index.vue]: Looks good overall, but there are two warnings if you run `yarn watch-vue-chrome` that would be good to address:

```
/enKrypt/packages/extension/src/providers/kadena/ui/send-transaction/components/send-alert.vue
  15:7  warning  'props' is assigned a value but never used  @typescript-eslint/no-unused-vars

/enKrypt/packages/extension/src/providers/kadena/ui/send-transaction/index.vue
  142:7  warning  'isPopup' is assigned a value but never used  @typescript-eslint/no-unused-vars
```

Comment (andborges) [reply packages/extension/src/providers/kadena/ui/send-transaction/index.vue]: Fixed :)

**Pull request merged**